runtime/src/main/java/org/corfudb/runtime/view/LayoutBuilder.java

```diff
@@ -21,6 +21,8 @@
 @Slf4j
 public class LayoutBuilder {
 
+    static public final int CHAIN_REPLICATION_MIN_NUM_NODES = 2;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)

Reply:
done

Reply:
done.